Tests/JWTKitTests/RSATests.swift

```diff
@@ -10,25 +10,24 @@ final class RSATests: XCTestCase {
     }
 
     func testRSADocs() async throws {
-        await XCTAssertNoThrowAsync(try await JWTKeyCollection().addRS256(key: .public(pem: publicKey)))
+        await XCTAssertNoThrowAsync(try await JWTKeyCollection().addRS256(key: RSA.PublicKey(pem: publicKey)))
     }
 
     func testPublicKeyInitialization() throws {
-        let rsaKey = try RSAKey(modulus: modulus, exponent: publicExponent)
-        XCTAssertNotNil(rsaKey.publicKey)
-        XCTAssertNil(rsaKey.privateKey)
+        let rsaKey = try RSA.PublicKey(modulus: modulus, exponent: publicExponent)
+        XCTAssertNotNil(rsaKey.backing)
```

Comment:
As mentioned before, don't test this. Use `XCTAssertNoThrow()` on the initializer.

Reply:
Whoops, leftover from before!